GitUI/CommandsDialogs/SettingsDialog/MergeToolsHelper.cs

```diff
@@ -402,8 +359,28 @@ private static string GetRegistryValue(RegistryKey root, string subkey, string k
             return value ?? string.Empty;
         }
 
-        private static string GetVisualStudioPath()
+        private static string GetVsDiffMergePath()
         {
+            //%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe
+            var vsVersionNumber = new string[] { "2020", "2019", "2018", "2017" };
+            var vsVersionType = new string[] { "Professional", "Community" };
```

Comment:
I add it to a new commit. It's really to please you because the good way to do that should have been to remove vsdiffmerge from the list if you are not on windows. But I let you do that if you have the time to do this intrusive developement for no added value and a lot of tech debt.

I remind you that I just fixed something broken when I was there to add a feature. My goal was not to refactore everthing and not see my PR merged....